ocaml/testsuite/tests/typing-modules/functors.ml

```diff
@@ -966,8 +943,8 @@ Error: Signature mismatch:
                    module E :
                      sig
                        module F :
-                         sig type x end -> sig type y end ->
-                           sig type z end -> sig type w end -> sig end
+                         sig ... end -> sig ... end -> sig ... end ->
```

Comment:
I don't really understand the objection. All these message have the form "These types don't match because these types don't match because these types don't match etc." Really, the only actionable part of the error message is the last one (way further below) which doesn't change with this PR. What useful information does the fully unfolded type of `C` provide in this particular spot?